fvm/blueprints/scripts/processScheduledCallbacksTransaction.cdc

```diff
@@ -0,0 +1,7 @@
+import "CallbackScheduler"
+
+transaction() {
+    execute {
+        CallbackScheduler.process()
```

Comment:
Its likely that this will need some input. 

The FVM will want to communicate how much computation effort is available.